src/I18nBundle/Helper/CookieHelper.php

```diff
@@ -33,8 +40,16 @@ public function get(Request $request, string $key = Definitions::REDIRECT_COOKIE
 
     public function set(Response $response, array $params): Cookie
     {
+        $cookieConfig = $this->configuration->getConfig('cookie');
+
+        $path = $cookieConfig['path'] ?? '/';
+        $domain = $cookieConfig['domain'] ?? null;
+        $secure = $cookieConfig['secure'] ?? false;
+        $httpOnly = $cookieConfig['httpOnly'] ?? true;
+        $sameSite = $cookieConfig['sameSite'] ?? Cookie::SAMESITE_LAX;
```

Comment:
Defaulting these values shouldn't be required if we can ensure them via configuration, right?

Reply:
Yes, that's correct. I've removed the default values from there.